workflows/push.yml

```diff
@@ -0,0 +1,12 @@
+on: push
```

Comment:
I am wondering, don't need for the `.github` folder? 🤔

Reply:
It should have been there, fixed, sorry :p